### DISCUSSION TITLE
Community Discord server

### DISCUSSION THREAD

**Discussion by yannickmonney** [tremorlabs/tremor-npm#390] [Ideas]
Hello, quick question, is there possibly a Discord server?

**Comment by mitrotasios** [comment]
there is a slack channel now [https://join.slack.com/t/tremor-community/shared_invite/zt-1u8jqmcmq-Fdr9B6MbnO7u8FkGh~2Ylg](https://t.co/g99pltTvIC) 🙂

**Reply by kempe-codelabs** [reply]
It's not possible to join this one if you don't have an @tremor.so email.

"The email address must match one of the domains listed below. Please try another email."
